packages/shared/src/components/sidebar/Sidebar.tsx

```diff
@@ -78,75 +77,110 @@ export default function Sidebar({
-          title: 'Popular',
-          path: '/popular',
-          action: () => onNavTabClick?.('popular'),
-        },
-        {
-          icon: <ListIcon Icon={UpvoteIcon} />,
-          title: 'Most upvoted',
-          path: '/upvoted',
-          action: () => onNavTabClick?.('upvoted'),
-        },
-        {
-          icon: <ListIcon Icon={DiscussIcon} />,
-          title: 'Best discussions',
-          path: '/discussed',
-          action: () => onNavTabClick?.('discussed'),
-        },
-        {
-          icon: <ListIcon Icon={SearchIcon} />,
-          title: 'Search',
-          path: '/search',
-          action: enableSearch ? () => enableSearch() : null,
-          hideOnMobile: true,
-        },
-      ],
+      icon: <ListIcon Icon={FilterIcon} />,
+      alert: alerts.filter && (
+        <AlertDot className="-top-0.5 -right-0.5" color={AlertColor.Fill} />
+      ),
+      title: 'Feed filters',
+      action: setLoaded,
+      hideOnMobile: true,
     },
     {
-      key: 'Manage',
-      items: [
-        {
-          icon: <ListIcon Icon={BookmarkIcon} />,
-          title: 'Bookmarks',
-          path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}bookmarks`,
-          hideOnMobile: true,
-          requiresLogin: true,
-        },
-        {
-          icon: <ListIcon Icon={EyeIcon} />,
-          title: 'Reading history',
-          path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}history`,
-          requiresLogin: true,
-        },
-        {
-          icon: <ListIcon Icon={SettingsIcon} />,
-          title: 'Customize',
-          action: () => setShowSettings(!showSettings),
-          active: showSettings,
-        },
-      ],
+      icon: <ListIcon Icon={HotIcon} />,
+      title: 'Popular',
+      path: '/popular',
+      action: () => onNavTabClick?.('popular'),
+    },
+    {
+      icon: <ListIcon Icon={UpvoteIcon} />,
+      title: 'Most upvoted',
+      path: '/upvoted',
+      action: () => onNavTabClick?.('upvoted'),
+    },
+    {
+      icon: <ListIcon Icon={DiscussIcon} />,
+      title: 'Best discussions',
+      path: '/discussed',
+      action: () => onNavTabClick?.('discussed'),
+    },
+    {
+      icon: <ListIcon Icon={SearchIcon} />,
+      title: 'Search',
+      path: '/search',
+      action: enableSearch ? () => enableSearch() : null,
+      hideOnMobile: true,
+    },
+  ];
+
+  const manageMenuItems: SidebarMenuItem[] = [
+    {
+      icon: <ListIcon Icon={BookmarkIcon} />,
+      title: 'Bookmarks',
+      path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}bookmarks`,
+      hideOnMobile: true,
+      requiresLogin: true,
+    },
+    {
+      icon: <ListIcon Icon={EyeIcon} />,
+      title: 'Reading history',
+      path: `${process.env.NEXT_PUBLIC_WEBAPP_URL}history`,
+      requiresLogin: true,
+    },
+    {
+      icon: <ListIcon Icon={SettingsIcon} />,
+      title: 'Customize',
+      action: () => setShowSettings(!showSettings),
+      active: showSettings,
```

Comment:
Now that we have separated our list for these items, I think we can take these two out and give them their own separate files then accept props to make it easier to manage.

I think in the future it will be easy to create a new set of sections and this one file would be filled so much by those lists. What do you think?

Reply:
They will be receiving a lot of props though.
Not sure if we want to pass down so much information vs keeping it there?

The list will not be re-used anywhere

Reply:
I agree with @sshanzel that this might be way easier to read. But if it feels like too much of a hassle at this point, we can leave it for the next one to pick it up whenever we need to touch this again.
This PR is already pretty massive so I understand if we don't want to make it even heavier

Reply:
Agreed. I think we can move forward with this.